L1Trigger/L1THGCal/interface/backend/HGCalSortingTruncationConfig_SA.h

```diff
@@ -0,0 +1,24 @@
+#ifndef __L1Trigger_L1THGCal_HGCalSortingTruncationConfig_SA_h__
+#define __L1Trigger_L1THGCal_HGCalSortingTruncationConfig_SA_h__
+
+#include <vector>
+
+namespace l1thgcfirmware {
+
+  class sortingTruncationAlgoConfig {
```

Comment:
Class names should start with a capital letter

Reply:
Changed